app/javascript/src/Common/application.ts

```diff
@@ -0,0 +1,65 @@
+import type { JQueryXHR } from 'Types/jquery/v1'
+
+const jQuery1 = window.$
+
+export default function (): void {
+  // disable links with 'data-disabled' attribute and display alert instead
+  // delegation on body fires before rails.js. FIXME: this is not a valid guard. If "data-disabled"
+  // removed manually from the DOM, the request will still go through. The link should be disabled
+  // (not clickable) and the server should return a flash error.
+  $('body').on('click', 'a[data-disabled]', (event) => {
+    event.stopPropagation()
```

Comment:
```suggestion
```

I understand that `stopImmediatePropagation` kinda includes `stopPropagation` (it does the same and more), so it's redundant to put both.

Reply:
My mistake 👍🏻